code/ATMOSPHERICS/components/components_base.dm

```diff
@@ -150,6 +150,9 @@ Helpers
 /obj/machinery/atmospherics/components/proc/update_parents()
 	for(DEVICE_TYPE_LOOP)
 		var/datum/pipeline/parent = PARENT_I
+		if(!parent)
+			PROCCRASH("Component is missing a pipenet! Rebuilding...")
+			build_network()
```

Comment:
This is how I've chosen to resolve the issue. I call PROCCRASH so that we get a stack trace and associated runtime information, but I also call build_network() to prevent further runtimes on this component. It's the best solution I've got for now.